qemu/accel/tcg/translate-all.c

```diff
@@ -1005,6 +1005,7 @@ static void tb_htable_init(struct uc_struct *uc)
 
 static void uc_tb_flush(struct uc_struct *uc) {
     tb_flush(uc->cpu);
+    tlb_flush(uc->cpu);
```

Comment:
Maybe another `UC_CTL_XXX`? The purpose of this `CTL` is slightly different I believe.

Reply:
Haven't got feedback to my comment https://github.com/unicorn-engine/unicorn/pull/1746#issuecomment-1434660135 so I have implemented the lacy solution. I'll update this.